dynamic_models/models.py

```diff
@@ -70,12 +71,9 @@ class FieldSchema(models.Model):
 
     name = models.CharField(max_length=63)
     model_schema = models.ForeignKey(ModelSchema, on_delete=models.CASCADE, related_name="fields")
-    data_type = models.CharField(
-        max_length=16, choices=FieldFactory.data_type_choices(), editable=False
-    )
     null = models.BooleanField(default=False)
```

Comment:
Any reason `null` is being treated specially here? Can we include it in `kwargs`?

Reply:
Because there were some checks on the null field, I thought we need it as a separate field (regarding that all Django model fields have a null argument). But now I changed it to a class property that gets the actual value from kwargs.